lib/web/scripts/oneoff/oneoff.sh

```diff
@@ -3,54 +3,64 @@ set -eu
 
 cdnBaseURL='{{.CDNBaseURL}}'
 teleportVersion='{{.TeleportVersion}}'
-teleportFlavor='{{.TeleportFlavor}}' # teleport or teleport-ent
+teleportArtifact='{{.TeleportArtifact}}' # teleport, teleport-ent or teleport-update
+teleportDirectory='{{.TeleportDirectory}}'
 successMessage='{{.SuccessMessage}}'
 entrypointArgs='{{.EntrypointArgs}}'
 entrypoint='{{.Entrypoint}}'
 packageSuffix='{{ if .TeleportFIPS }}fips-{{ end }}bin.tar.gz'
 fips='{{ if .TeleportFIPS }}true{{ end }}'
+supportedOSes='{{join .SupportedOSes " "}}'
```

Comment:
Can we pass the expected value instead of converting it using the template engine?
We should keep this shell script as simple as possible